webapp/src/blocks.tsx

```diff
@@ -159,6 +160,9 @@ export class Editor extends toolboxeditor.ToolboxEditor {
             pxt.Util.toArray(document.querySelectorAll(classes)).forEach((el: HTMLElement) => el.style.display = 'none');
             if (this.editor) Blockly.hideChaff();
             if (this.toolbox) this.toolbox.clearExpandedItem();
+            // isFirstLoad can be true and isFirstBlocklyLoad can be false
+            // if the app switches to JavaScript or Python immediately.
+            this.isFirstLoad = false;
```

Comment:
A better way to achieve this would be welcome.

Reply:
As discussed, I can't see a better way but it does appear that isFirstBlocklyLoad is never read, so we can at least make it less confusing by removing that.

Reply:
Removed `isFirstBlocklyLoad ` here [ef98b65](https://github.com/microbit-matt-hillsdon/pxt/pull/2/commits/ef98b65e39345a2e434253848b277ac705aa9d5f)